mxabierto/guia
# Add resource 'Consejo Institucional' to Templates section

## How to Test

1. Go to 'Manuales y Plantillas' section
2. Check for this

![validate- 44](https://cloud.githubusercontent.com/assets/2633527/8887045/1f438948-323a-11e5-82c3-fbadedf15a80.png)
